clients/src/main/java/org/apache/kafka/clients/consumer/internals/AsyncKafkaConsumer.java

```diff
@@ -1808,12 +1810,23 @@ private boolean processBackgroundEvents() {
      * @param future         Event that contains a {@link CompletableFuture}; it is on this future that the
      *                       application thread will wait for completion
      * @param timer          Overall timer that bounds how long to wait for the event to complete
+     * @param ignoreErrorEventException When processing background events, some error events are not triggered by current call.
+     *                                  ignoreErrorEventException is used to filter out these exceptions. For example,
+     *                                  if users subscribe to an invalid topic name, there will have some error events,
+     *                                  because network thread keep trying to send MetadataRequest in the background.
+     *                                  When users want to unsubscribe it, ignoreErrorEventException should be used to ignore previous error.
```

Comment:
Super nit-picky: can we update the other `@params`’ descriptions so they're vertically aligned?

Reply:
Yes, updated it. Thanks for the suggestion. 👍